windows/progress-view/ProgressViewView.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ProgressViewView.h"
+#include "ProgressViewView.g.cpp"
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::Foundation::Metadata;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Xaml;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Input;
+    using namespace Windows::UI::Xaml::Media;
+    using namespace Windows::UI::Xaml::Media::Imaging;
+} // namespace winrt
+
+namespace winrt::progress_view::implementation {
+
+    ProgressViewView::ProgressViewView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        this->Minimum(0);
```

Comment:
are these hardcoded values what the module specifies as default?

Reply:
The module specifies the progress value to be between 0 and 1. By default the min/max of XAML progress bar is 0/100 but I left it in for readability. Would it be better to take it out?

Reply:
Not sure about this one. If these are the default XAML values, you don't need to set them explicitly yourself. 
If you want to keep the XAML defaults, you can a comment saying what they are. Still, it's probably better to put it as a default prop value on the JS side. You should also consider, if the default should be the same as for other platforms (as defined by this module) or whether we'd like the XAML one to override it. It's q question you could ask the maintainers when they review this PR.
Perhaps @asklar could help, as well.